README.md

```
@@ -23,3 +23,38 @@ make dev
 ```shell
 poetry run scrapy runspider contrib/scraper_natal.py -o natal.jsonl
 ```
+
+#### WhatsApp Automation
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)